tests/js/spec/components/acl/role.spec.jsx

```diff
@@ -91,6 +91,24 @@ describe('Role', function () {
       ConfigStore.config.user = user;
     });
 
+    it('updates if user changes', function () {
+      const user = {...ConfigStore.config.user};
+      ConfigStore.config.user = undefined;
+      const {rerender} = mountWithTheme(<Role role="member">{childrenMock}</Role>, {
+        context: routerContext,
+      });
+
+      expect(childrenMock).toHaveBeenCalledWith({
+        hasRole: false,
+      });
+      ConfigStore.config.user = user;
+
+      rerender(<Role role="member">{childrenMock}</Role>);
+      expect(childrenMock).toHaveBeenCalledWith({
```

Comment:
Added to be sure that useMemo properly updates bc I wasnt sure about the user reference